src/components/Card/card.module.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+    border: 2px solid black;
+    border-radius: 10px;
+    padding: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin-bottom: 2rem;
+    cursor: pointer;
+  }
+  
```

Comment:
@ASHWINIAJIT Can you please remove these whitespaces?

Reply:
Removed all white spaces from CSS file.